internal/build/view.go

```diff
@@ -16,7 +16,10 @@ func BuildSummary(build *buildkite.Build) string {
 		buildCreator(build),
 		build.CreatedAt.UTC().Format(time.RFC1123Z))
 	hash := *build.Commit
-	commitDetails := fmt.Sprintf("Branch: %s / Commit: %s \n", *build.Branch, hash[0:8])
+	if len(hash) > 8 {
```

Comment:
Should this instead be:
```suggestion
	if len(hash) >= 7 {
```